README.md

```diff
@@ -70,7 +66,14 @@ $ sudo ln -s posix_c.so /usr/lib/x86_64-linux-gnu/lua/5.1/posix.so
 *Note 2 - only needed if optional man pages are to be created.  Only one
 of asciidoc or asciidoctor is needed.  Asciidoc is used if both are installed.*
 
-*Note 3 - optional, for enabling additional tests*.
+The following optional dependencies enable additional testing:
+
+**redhat**        | **ubuntu**        | **version**
+----------        | ----------        | -----------
+aspell            | aspell            |
+valgrind          | valgrind          |
+mpich-devel       | libmpich-dev      |
```

Comment:
Looks like the copy-paste line further down for ubuntu did not get the `mpich` -> `libmpich-dev` update.

Reply:
Doh!